src/tailwind/defaultConfig.ts

```diff
@@ -362,9 +364,13 @@ export default {
     |*/
 
     maxHeight: {
+      screen: "100vh",
       full: "100%",
       dropdown: "240px",
+      image: "180px",
+      imageLarge: "330px",
       dropdownSM: "190px",
+      imageInfo: "400px",
```

Comment:
🤔 can we keep these sizes in the projects? they're not referenced within the image component at all. it feels like a breach of separation of concerns to me. the component library should not know anything about the sizes used in projects and should be flexibly working with any

even if it's just about sharing common sizes, I'd keep them in the projects in this case as it's a layout concern, not a component concern

Reply:
I get what you mean, I removed them and I will adjust the tailwind config directly in listings and DH